### DISCUSSION TITLE
Australia Post (AusPost) using AppIntro

### DISCUSSION THREAD

**Discussion by irresistibl** [AppIntro/AppIntro#1176] [General]
Hi,
I recently spotted that the [Australia Post mobile application on Android](https://play.google.com/store/apps/details?id=au.com.auspost.android) is using and also directly references your git here in their about section.

Given you've got a section acknowledging apps that are using this, I figured this would be a nice inclusion to that 😊

**Comment by cortinico** [comment]
Feel free to send a PR against the README 👍
